packages/account/src/Components/forms/scroll-to-field-with-error.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useFormikContext } from 'formik';
+
+type TScrollToFieldWithError = {
+    fields_to_scroll_top?: string[];
+    fields_to_scroll_end?: string[];
+    should_recollect_inputs_names?: boolean;
+};
+
+const ScrollToFieldWithError = ({
+    fields_to_scroll_top,
+    fields_to_scroll_end,
+    should_recollect_inputs_names = false,
+}: TScrollToFieldWithError) => {
+    const [all_page_inputs_names, setAllPageInputsNames] = React.useState<string[]>([]);
+    const formik = useFormikContext();
```

Comment:
```suggestion
    const {isSubmitting: is_submitting, errors} = useFormikContext();
```
what if we desctructure it like this?